client/js/components/statement/assessmentTable/DpMoveStatementModal.vue

```diff
@@ -8,113 +8,111 @@
-          <select
-            id="r_target_procedure"
-            name="r_target_procedure"
-            class="width-100p u-mb"
-            v-model="selectedProcedureId">
-            <option value="">
-              -
-            </option>
-            <option
-              v-for="procedure in availableProcedures"
-              :key="procedure.id"
-              :value="procedure.id">
-              {{ procedure.name }}
-            </option>
-          </select>
-          <div
-            v-if="hasPermission('feature_statement_content_changes_view') || hasPermission('feature_statement_content_changes_save')"
-            class="u-mb">
+        <!-- When both permissions are available, the user is prompted to choose which type of procedure she wants to move the statement to -->
+        <template v-if="hasPermission('feature_statement_move_to_foreign_procedure')">
+          <label class="u-mb-0_5 inline-block">
             <input
-              type="checkbox"
-              id="deleteVersionHistory"
-              v-model="deleteVersionHistory"
-              aria-describedby="deleteHistoryDesc">
-            <label
-              for="deleteVersionHistory"
-              class="inline-block u-mb-0">{{ Translator.trans('delete.history') }}</label>
-            <p
-              class="lbl__hint"
-              id="deleteHistoryDesc">
-              {{ Translator.trans('delete.history.description') }}
-            </p>
-          </div>
-          <!-- The button disabled-attribute is set to true when the user is not the assignee of all fragments or if any fragments are assigned to departments -->
-          <button
-            type="button"
-            class="btn btn--primary float-right"
-            @click.prevent.stop="moveStatement"
-            :disabled="!userIsAssigneeOfAllFragments || !fragmentsAreNotAssignedToDepartments">
-            {{ Translator.trans('statement.moveto.procedure.action') }}
-          </button>
+              type="radio"
+              name="procedure_permissions"
+              v-model="procedurePermissions"
+              value="accessibleProcedures"
+              required> {{ Translator.trans('procedure.accessible') }}
+          </label>
+          <label class="u-mb-0_5 u-ml inline-block">
+            <input
+              type="radio"
+              name="procedure_permissions"
+              v-model="procedurePermissions"
+              value="inaccessibleProcedures"> {{ Translator.trans('procedure.inaccessible') }}
+          </label>
         </template>
-      </div>
-    </dp-modal>
-  </portal>
+
+        <label
+          class="u-mb-0"
+          for="r_target_procedure">{{ Translator.trans('statement.moveto.procedure.target') }}</label>
+        <select
+          id="r_target_procedure"
+          name="r_target_procedure"
+          class="width-100p u-mb"
+          v-model="selectedProcedureId">
+          <option value="">
+            -
+          </option>
```

Comment:
The truth is that I only removed the `<portal>` tag wrapping the component because the goal here is to make things work in vue 3, so I'd suggest to leave the rest of the code as it was

Reply:
Agree, this was just something with a origin that was not obvious for me.